lib/vty.c

```diff
@@ -312,6 +312,13 @@ int vty_json_no_pretty(struct vty *vty, struct json_object *json)
 	return vty_json_helper(vty, json, JSON_C_TO_STRING_NOSLASHESCAPE);
 }
 
+void print_empty_json_obj(struct vty *vty)
```

Comment:
Let's use the same naming convention, something like: `vty_json_empty()` or so.

Reply:
Will change the name.

Reply:
Fixed it